docs/ui/auth/fragments/react/sign-out.md

```diff
@@ -1,4 +1,4 @@
-<amplify-sign-out button-text="Custom Text"></amplify-sign-out>
+<amplify-sign-out buttonText="Custom Text"></amplify-sign-out>
```

Comment:
It's also `AmplifySignOut` instead of `amplify-sign-out` too, right?

```suggestion
<AmplifySignOut buttonText="Custom Text"></AmplifySignOut>
```